drivers/builtin/src/pk.c

```diff
@@ -993,11 +993,11 @@ int mbedtls_pk_verify_ext(mbedtls_pk_type_t type, const void *options,
         return MBEDTLS_ERR_PK_BAD_INPUT_DATA;
     }
 
-    if (!mbedtls_pk_can_do(ctx, type)) {
+    if (!mbedtls_pk_can_do(ctx, (mbedtls_pk_type_t) type)) {
```

Comment:
Is there any point in introducing a static inline function to do this conversion? Maybe we'll want to instrument it later? Or does it just not matter since this is a cast to a supertype?